cmd/adapter-rest/startcmd/start.go

```diff
@@ -84,6 +92,26 @@ const (
 	modeFlagUsage = "Mode in which the edge-adapter service will run. Possible values: " +
 		"['issuer', 'rp']."
 	modeEnvKey = "ADAPTER_REST_MODE"
+
+	// inbound host url flag
+	didCommInboundHostFlagName  = "didcomm-inbound-host"
+	didCommInboundHostEnvKey    = "ADAPTER_REST_DIDCOMM_INBOUND_HOST"
+	didCommInboundHostFlagUsage = "Inbound Host Name:Port. This is used internally to start the didcomm server." +
+		" Alternatively, this can be set with the following environment variable: " + didCommInboundHostEnvKey
+
+	// inbound host external url flag
+	didCommInboundHostExternalFlagName  = "didcomm-inbound-host-external"
+	didCommInboundHostExternalEnvKey    = "ADAPTER_REST_DIDCOMM_INBOUND_HOST_EXTERNAL"
+	didCommInboundHostExternalFlagUsage = "Inbound Host External Name:Port." +
+		" This is the URL for the inbound server as seen externally." +
+		" If not provided, then the internal inbound host will be used here." +
+		" Alternatively, this can be set with the following environment variable: " + didCommInboundHostExternalEnvKey
+
+	// db path
+	didCommDBPathFlagName  = "didcomm-db-path"
```

Comment:
not needed we will use sql storage in the future

Reply:
this flag needed just for leveldb

Reply:
SQL store is not available at the moment. Created an issue to switch - https://github.com/trustbloc/edge-adapter/issues/42.